scripts/config/functions.cfg

```diff
@@ -0,0 +1,252 @@
+                  echo -e 
+        fi
+}
+
+function build_monitor {
+  #Build the monitor from latest tag
+  echo -e
+  echo -e "${GREEN}Building your monitor binary...${NC}"
+  echo -e
+
+  #If OVERRIDE_VER is set SHOWVER should take it into account
+  [[ ! -z "$OVERRIDE_VER" ]] && SHOWVER=$(echo ${OVERRIDE_VER#${TMPVAR}})
+  [[ ! -z "$OVERRIDE_VER" ]] && VERSION=$(echo $OVERRIDE_VER)
+
+  #Building the node from the local directory
+  cd $WORKDIR
+  echo -e "${GREEN}Compiling binary with version ${CYAN}$VERSION${GREEN}...${NC}"
+  echo -e
+
+  cd cmd/monitor
+  # alter/create the local.go file that is .gitignored
+  appVersion=$SHOWVER-0-$(git describe --tags --long | tail -c 11)
+  echo "package main
+
+  func init() {
+  	appVersion = \"$appVersion\"
+  }" > local.go
+
+  git fetch
+  git checkout $VERSION
+  if [ "$VERSION" == "$SHOWVER" ]; then
+    # we are on a branch, we can do a git pull
+    git pull
+  fi
+
+  go build
+  echo -e "${GREEN}Done !${NC}"
+}
+
+function show_menu {
+echo -e
+echo -e
+echo -e "${CYAN}MultiversX ${RED}$ATTENTION${CYAN} scripts options:${NC}"
+echo -e
+echo -e "${GREEN} 1) ${CYAN}init{GREEN} - init the scripts by creating the config/local.cfg file${NC}"
+echo -e "${GREEN} 2) ${CYAN}install${GREEN} - Regular install process for the monitor${NC}"
+echo -e "${GREEN} 3) ${CYAN}upgrade${GREEN} - Run the upgrade process for the monitor${NC}"
+echo -e "${GREEN} 4) ${CYAN}start${GREEN} - Start the monitor${NC}"
+echo -e "${GREEN} 5) ${CYAN}stop${GREEN} - Stop the monitor${NC}"
+echo -e "${GREEN} 6) ${CYAN}cleanup${GREEN} - Remove everything from the host${NC}"
+echo -e "${GREEN} 7) ${CYAN}get_logs${GREEN} - Get the logs from the monitor${NC}"
+echo -e "${GREEN} 8) ${CYAN}quit${GREEN} - Exit this menu${NC}"
+echo -e
+echo -e
+}
+
+function systemd {
+  #Add systemd monitor service & syslogd logging
+  echo -e
+  echo -e "${GREEN}Installing systemd unit for the monitor...${NC}"
+  echo -e
+
+  #Create the service file for our node
+  echo "[Unit]
+  Description=MultiversX keys monitor
+  After=network-online.target
+
+  [Service]
+  User=$CUSTOM_USER
+  WorkingDirectory=$WORKDIR/cmd/monitor
+  ExecStart=$WORKDIR/cmd/monitor/monitor -log-level *:DEBUG $MONITOR_EXTRA_FLAGS
+  StandardOutput=journal
+  StandardError=journal
+  Restart=always
+  RestartSec=3
+  LimitNOFILE=4096
+
+  [Install]
+  WantedBy=multi-user.target" > mx-chain-keys-monitor.service
+
+  #Move files to appropriate locations
+  if [ -f "/etc/systemd/system/mx-chain-keys-monitor.service" ]; then
+    echo -e
+    echo -e "${GREEN}Refreshing mx-chain-keys-monitor service (stop-disable-remove)...${NC}"
+    echo -e
+    sudo systemctl stop mx-chain-keys-monitor
+    sleep 1
+    sudo systemctl disable mx-chain-keys-monitor.service
+    sleep 1
+    sudo rm -f /etc/systemd/system/mx-chain-keys-monitor.service
+    sleep 1
+  fi
+  sudo mv mx-chain-keys-monitor.service /etc/systemd/system/
+  sudo systemctl enable mx-chain-keys-monitor.service
```

Comment:
Maybe add a "sudo systemctl daemon-reload" command (in case the params added by the $MONITOR_EXTRA_FLAGS variable have changed ?

Reply:
added before this call